editor/selectors.js

```diff
@@ -715,3 +715,42 @@ export function getRecentlyUsedBlocks( state ) {
 	// resolves the block names in the state to the block type settings
 	return state.userData.recentlyUsedBlocks.map( blockType => getBlockType( blockType ) );
 }
+
+/**
+ * Returns the peerData Object
+ *
+ * @param {Object} state Global application state
+ * @param {Object} uid current props uid
+ *
+ * @return {Object} Name, Css style for peerColor, peerID
+ */
+export function getPeerData( state, uid ) {
+	const { peerID, peerName, peerColor, blocksByUid } = state.collaborationMode;
+
+	if ( peerColor && peerID && peerName && peerID !== window.grtcProps.peerID && blocksByUid === uid ) {
```

Comment:
It's a bit weird to use global variables in a selector? What's the purpose of the `window.grtcProps` ? Can't we move this to the store maybe?